scripts/test-dist.js

```diff
@@ -20,7 +20,12 @@ shell.exec("npm init -y", { cwd: tmpDir });
 shell.exec(`npm install "${tarPath}"`, { cwd: tmpDir });
 shell.config.silent = false;
 
-const cmd = `yarn test --color --runInBand ${
+const reporters =
+  +process.version.match(/^v\d+\./)[1] >= 6
+    ? "--reporters default --reporters jest-junit"
+    : "";
+
+const cmd = `yarn test --color --runInBand ${reporters} ${
```

Comment:
Why not add another conditional?

```js
${process.env.JEST_JUNIT ? "--reporters ..." : ""}
```

And then just set this env in `test_prod_node9`